mmv1/templates/terraform/examples/google_storage_managed_folder.tf.erb

```diff
@@ -0,0 +1,10 @@
+resource "google_storage_managed_folder" "managed_folder" {
+  bucket = google_storage_bucket.bucket.name
+  name   = "managed-folder-name"
+}
+
+resource "google_storage_bucket" "bucket" {
+  name     = "<%= ctx[:vars]['bucket_name'] %>"
+  location = "me-central2"
```

Comment:
This seems to be causing the tests to fail - is there a particular reason it needs to be this region? Otherwise most tests use `US`.

```suggestion
  location = "US"
```

Reply:
Solved in last commit